src/Hit/Git.hs

```diff
@@ -86,13 +86,25 @@ runAmend :: IO ()
 runAmend = do
     "git" ["add", "."]
     "git" ["commit", "--amend", "--no-edit"]
-    runPush True
+    runPush True False
 
 -- | @hit push@ command.
-runPush :: Bool -> IO ()
-runPush isForce = getCurrentBranch >>= \branch ->
-    "git" $ ["push", "--set-upstream", "origin", branch]
-         ++ ["--force" | isForce]
+runPush :: Bool -> Bool -> IO ()
+runPush isForce assignOwner = do
+    branch <- getCurrentBranch
+    "git" $ ["push", "--set-upstream", "origin", branch] ++ ["--force" | isForce]
+    when assignOwner (attemptAssignment branch)
+    where attemptAssignment :: Text -> IO ()
+          attemptAssignment branch = do
+            let issue = mkIssueId <$> issueFromBranch branch
+            case issue of
+              Nothing ->  errorMessage "Unable to parse Issue ID from Branch"
+              Just issueId -> do
+                assignmentResults <- assignToIssue issueId
+                either printError finish assignmentResults
+          printError :: Show a => a -> IO ()
+          printError = errorMessage . T.pack . show
+          finish = const $ return ()
```

Comment:
I guess it's better to print info if the issue was assigned to the person.

Reply:
Very true. Thank you.